apps/portal/src/app/tokens/deploy-erc1155/page.mdx

```diff
@@ -0,0 +1,16 @@
+import { Step, Steps, DocImage, createMetadata } from "@doc";
+import TokenDeployment from "../assets/token_deployment.png";
+
+export const metadata = createMetadata({
+    image: {
+        title: "Deploy an ERC-1155 token",
+        icon: "tokens",
```

Comment:
`icon: "tokens"` is not valid. we don't have tokens icon image. 
The valid icon names are following: 

<img width="826" height="608" alt="image" src="https://github.com/user-attachments/assets/79df7033-8a91-4b15-8c7b-981ded49bcc0" />